include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -791,6 +791,60 @@ The data will be distributed between threads as follows:
   let hasCustomAssemblyFormat = 1;
 }
 
+def WmmaEncodingAttr : DistributedEncoding<"WmmaEncoding", "wmma_encoding", [MmaEncodingTrait]> {
```

Comment:
Can we pick a more explicit name with AMD "namespacing" in it? (like `NvidiaMmaEncoding`).